CMSIS/NN/Include/arm_nnfunctions.h

```diff
@@ -470,6 +483,7 @@ extern    "C"
    * @param[in]      buffer_a  pointer to buffer space used for input optimization(partial im2col) and is necessary
```

Comment:
Same remark for param[in]. param[out] ?

I won't add the same comment for other similar lines...